src/js/components/Skeleton/propTypes.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+
+import {
+  genericProps,
+  heightPropType,
+  padPropType,
+  roundPropType,
+  skeletonColorsPropType,
+  widthPropType,
+} from '../../utils/general-prop-types';
+
+let PropType = {};
+if (process.env.NODE_ENV !== 'production') {
+  PropType = {
+    ...genericProps,
+    as: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.func,
+      PropTypes.element,
+    ]),
+    colors: skeletonColorsPropType,
+    height: heightPropType,
```

Comment:
margin and message prop

Reply:
Wouldn't expect message here as I noted above. Margin is in the genericProps